docs/concepts/service-catalog/index.md

```diff
@@ -0,0 +1,233 @@
+* `ServiceClass`: A Managed Service offered by a particular Service Broker.
+When a new `ServiceBroker` resource is added to the cluster, the Service Catalog controller connects to the Service Broker to obtain a list of available Managed Services. It then creates a new `ServiceClass` resource corresponding to each Managed Service.
+* `ServiceInstance`: A provisioned instance of a `ServiceClass`.
+These are created by Cluster Operators to make a specific instance of a Managed Service available for use by one or more in-cluster applications.
+When a new `ServiceInstance` resource is created, the Service Catalog controller will connect to the appropriate Service Broker and instruct it to provision the service instance.
+* `ServiceBinding`: Access credentials to a `ServiceInstance`.
+These are created by Cluster Operators who want their applications to make use of a Service `ServiceInstance`.
+Upon creation, the Service Catalog controller will create a Kubernetes `Secret` containing connection details and credentials for the Service Instance, which can be mounted into Pods.
+
+### Mutual TLS encryption
+
+The mutual TLS protocol encrypts communication between the Service Catalog extension API server and the main Kubernetes API server. 
+
+During installation, Service Catalog creates its own certificate authority (CA), and generates its own public and private keys, signed by this CA.
+The CA public certificate is installed into the main API server when Service Catalog registers its API.
+Service Catalog accesses the main API server CA certificate from the API server ConfigMap, after being granted the *extension-apiserver-authentication-reader* role. 
+
+Certificate rotation is handled by exposing a Service Catalog URL, similar to a `/statusz` endpoint, which returns the number of days until the CA and certificates expire.
+Alerts for certificate rotation can be created by monitoring this URL.
+
+### Authentication
+
+Service Catalog supports these methods of authentication: 
+
+* Basic (username/password)
+* [OAuth 2.0 Bearer Token](https://tools.ietf.org/html/rfc6750)
+
+## Usage
+
+The {% glossary_tooltip text="Cluster Operator" term_id="cluster-operator" %} can use the Service Catalog API Resources to provision Managed Services and make them available within the Kubernetes cluster. The steps involved are:
+
+1. Listing the Managed Services available from a Service Broker.
+1. Provisioning a new instance of the Managed Service.
+1. Binding to the Managed Service, which returns the connection credentials.
+1. Mapping the connection credentials into the application.
+
+### Listing Managed Services
+
+First, the Cluster Operator must create a `ServiceBroker` resource within the `servicecatalog.k8s.io` group. This resource contains the URL and connection details necessary to access a Service Broker endpoint.
```

Comment:
Use the Cluster Operator variable here? Either at first use or everywhere.

Reply:
Ok, I'm switching all of them to use the hover-over tooltip. I'm doing that for Application Developer as well.